openlibrary/plugins/worksearch/tests/test_worksearch.py

```diff
@@ -1,9 +1,32 @@
-from __future__ import print_function
 import unittest
-from openlibrary.plugins.worksearch.code import read_facets, sorted_work_editions, parse_query_fields, escape_bracket, run_solr_query, get_doc, build_q_list, escape_colon, parse_search_response
+from openlibrary.plugins.worksearch.code import (
+    read_facets, sorted_work_editions, parse_query_fields,
+    escape_bracket, run_solr_query, get_doc, build_q_list,
+    escape_colon, parse_search_response
+)
+from openlibrary.core import search as ia_search
```

Comment:
Let's create a separate file for these tests. The pattern is one `test_` file for every python file.

Reply:
Agree -- these test won't be getting run otherwise.